content/compose/compose-file/build.md

```diff
@@ -496,7 +515,7 @@ tags:
 
 ### ulimits
 
-> Available with Docker Compose version 2.24.0 and later.
+{{< introduced compose 2.23.1 "../release-notes.md#2231" >}}
```

Comment:
This is the only version note in this file I saw in the [deploy preview](https://deploy-preview-19773--docsdocker.netlify.app/compose/compose-file/build/) for some reason.